### ISSUE TITLE
Max Payload size for WAF and logging (Filebeat, et al)

### ISSUE THREAD

**Issue by tzuryby** [curiefense/curiefense#524]
**Large body requests**

### Large body requests should:
1. Skip body inspection
2. rendered in a way that they can be logged.

** we Should maintain a paremter per deployment (global is fine, no need for WAF policy level) that will set the threshold for this.

![image](https://user-images.githubusercontent.com/7790/127209560-42f995e1-bda6-4890-981e-993167f3c210.png)

**Comment by bartavelle** [comment]
Where should this be configured? What does "rendered in a way that they can be logged" mean? If it means dumping the body to the log system, I think this should be configurable as it might clog the whole thing.

**Comment by bartavelle** [comment]
This needs bumping!

**Comment by bartavelle** [comment]
(I am working on related topics, so it would be nice to decide how this is to be configured)

**Comment by techiek7** [comment]
Any progress/update here?

**Comment by bartavelle** [comment]
Yes! It is implemented in the filter, but needs UI support and testing :)

**Comment by techiek7** [comment]
ok, any way to configure it via confserver API ?

**Comment by bartavelle** [comment]
Not yet, but there should be, soon.
